src/TorchSharp/TorchVision/Image.cs

```diff
@@ -0,0 +1,177 @@
+using System.IO;
+using System.Threading.Tasks;
+
+using static TorchSharp.torch;
+
+namespace TorchSharp.torchvision
+{
+    public static partial class io
+    {
+        /// <summary>
+        /// <cref>Imager</cref> to be used when a <cref>torchvision.io</cref> image method's <c>imager</c> is unspecified.
+        /// </summary>
+        public static Imager DefaultImager { get; set; }
```

Comment:
By removing all implementations, this could be null. Would we be better off with a skeleton implementation that just throws an exception, but one that is a little bit more helpful than 'NullReferenceException'?